cmd/frontend/graphqlbackend/search_alert.go

```diff
@@ -443,3 +443,23 @@ func addQueryRegexpField(query *query.Query, field, pattern string) syntax.Parse
 	}
 	return expr
 }
+
+// newAlert creates a simple alert with a title and description that can be
+// directly returned by functions that implement the GraphQL search
+// interface.
+func newAlert(title string, description string) SearchImplementer {
```

Comment:
nit: It's in the huge namespace of "graphqlbackend", better to be more explicit.
```suggestion
func newSearchAlert(title string, description string) SearchImplementer {
```